app/Http/Controllers/Api/AssetsController.php

```diff
@@ -670,8 +670,6 @@ public function store(StoreAssetRequest $request): JsonResponse
             }
 
             return response()->json(Helper::formatStandardApiResponse('success', $asset, trans('admin/hardware/message.create.success')));
-
-            return response()->json(Helper::formatStandardApiResponse('success', (new AssetsTransformer)->transformAsset($asset), trans('admin/hardware/message.create.success')));
```

Comment:
Please put this back in - it was there because that's the eventual shape of the API